tests/tests_fabric/plugins/precision/test_bitsandbytes.py

```diff
@@ -264,3 +265,10 @@ def forward(self, x):
     assert model.linear.weight.shape == (128, 1)
     # Shapes match during forward (weight is being dequantized during forward)
     model(torch.randn(2, 16, device=fabric.device))
+
+    # Test with lazy load (LitGPT uses this)
+    # TODO: Replace `_lazy_load` with `torch.load(..., mmap=True)` in LitGPT
+    state_dict = _lazy_load(tmp_path / "checkpoint.pt")
```

Comment:
how about using steps for these long tests?
https://smarie.github.io/python-pytest-steps/